Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:16-alpine
```

Comment:
You are using Node.js 16.20.2. For Next.js, Node.js version >= v18.18.0 is required.

Reply:
Ok, será alterado.